starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const router  = express.Router();
+const CelebModel = require('../models/celeb.model')
+const bodyParser = require('body-parser');
+
+/* GET home page */
+router.get('/celebrities/index', (req, res, next) => {
+  CelebModel.find()
+    .then((celebs) => {
+      res.render('celebrities/index.hbs', {celebs})
+    })
+    .catch((err)=>{console.log('error in home page creation', err)})
```

Comment:
when writing code statements in one line you don't need the oute curly brackets
`.catch((err) => console.log('error in home page creation', err))`